docs/_build/html/_sources/quickstart.rst.txt

```diff
@@ -0,0 +1,180 @@
+    tt_a = ttax.random_.matrix(rng1, (left_shape, sum_shape), tt_rank=3, dtype=dtype)
+    tt_b = ttax.random_.matrix(rng2, (sum_shape, right_shape), tt_rank=[1, 4, 3, 1],
+                               dtype=dtype)
+    res_actual = ttax.full(ttax.ops.matmul(tt_a, tt_b))
+    res_desired = ttax.full(tt_a) @ ttax.full(tt_b)
+    np.testing.assert_allclose(res_actual, res_desired, 1e-3)
+    
+Working with batches
+--------------------
+
+TTAX tries to support the work with multidimensional batches of tensors where it is possible, taking the input of multidimensional batches as if they were taking ordinary tensors. It means that if A and B are batches of TT-tensors/matices you can do A+B like you do for TT-tensors/matrices.
+
+Let's see how it works. We create 2 batches of TT-tensors of the same batch size and then compare the result of sum in TT format with the one in full format.
+
+.. code-block:: python
+
+    rng1, rng2 = jax.random.split(jax.random.PRNGKey(0))
+    dtype = jnp.float32
+    tt_a = ttax.random_.tensor(rng1, (2, 1, 3, 4), tt_rank=2, batch_shape=(3,),
+                      dtype=dtype)
+    tt_b = ttax.random_.tensor(rng2, (2, 1, 3, 4), tt_rank=[1, 2, 4, 3, 1],
+                      batch_shape=(3,), dtype=dtype)
+    res_actual = ttax.ops.full(tt_a + tt_b)
+    res_desired = ttax.ops.full(tt_a) + ttax.ops.full(tt_b)
+    np.testing.assert_allclose(res_actual, res_desired, 1e-6)
+    
+.. note:: 
+   
+   You can use both tensor indexing and batch indexing.
+
+You can use tensor indexing to get specified element / slice.
+
+.. code-block:: python
+
+    rng = jax.random.PRNGKey(41)
+    tt = ttax.random_.tensor(rng, [2, 3, 4])
+    print(tt[1, :, :].shape, "<- 2D Tensor-Train")
+    print(tt[1:2, :, :].shape, "<- 3D Tensor-Train")
+    
+Similar idea for batch indexing but with a slightly different syntax.
+
+.. code-block:: python
+    
+    rng = jax.random.PRNGKey(41)
+    tt = ttax.random_.tensor(rng, [2, 3, 4], tt_rank=2, batch_shape=(3, 3, 3,))
+    tt.batch_loc[1, :, :]
+    
+Speeding up your code
+---------------------
+
+Our library is written with the expectation of using the jax.jit for acceleration.
+
+Some routines were based on einsum (see TTEinsum), to speed them up you can use fuse method (see compile.fuse).
+
+Below is the example of how to use such speeding up and the difference it provides.
+
+.. code-block:: python
+
+    rng = jax.random.PRNGKey(42)
+    tt_a = ttax.random_.tensor(rng, [3, 4, 5], tt_rank=30)
+    tt_b = ttax.random_.tensor(rng, [3, 4, 5], tt_rank=30)
+    tt_c = ttax.random_.tensor(rng, [3, 4, 5], tt_rank=1)
+    
+.. code-block:: python
+    
+    def f(a, b, c):
+      return ttax.flat_inner(ttax.multiply(a, b), c)
+      
+.. code-block:: python
+
+    fused_f = ttax.fuse(f)
+    jit_f = jax.jit(f)
+    jit_fused_f = jax.jit(fused_f)
+    
+.. code-block:: python
+
+    %timeit f(tt_a, tt_b, tt_c)
+    %timeit jit_f(tt_a, tt_b, tt_c)
+    %timeit jit_fused_f(tt_a, tt_b, tt_c)
+    
+
```

Comment:
I don't think you need to commit the HTML folder (the files below). You can add it to git ignore to avoid commiting by accident